Use a packaged LLVM 3.3 version

jasuarez opened the pull request:
Unfortunately, LLVM 3.3 is not packaged for Ubuntu Xenial. And
porting from old Ubuntu Trusty is causing lot of troubles.

So we were just building LLVM 3.3 from scratch. But this takes time.
Fortunately, we can package that manually-built version directly using
checkinstall.

This commit just uses LLVM 3.3 from a package created with checkinstall.
It contains the same version as if it were build manually.

This fixes issue #15.

tanty commented:
Couple of comments:
- I would like to see how the checkinstall .deb file is generated.
- Just remove everything that you are commenting.

jasuarez commented:
- Checkinstall is run interactively. So I just filled the proper fields when required
- I've left the comment to know how the package is built (checkinstall is run only in the make install process).

In any case, I'll try to provide a way of running checkinstall non-interactively, and paste as comment how to re-build the package.

jasuarez committed:
Use a packaged LLVM 3.3 version

Unfortunately, LLVM 3.3 is not packaged for Ubuntu Xenial. And
porting from old Ubuntu Trusty is causing lot of troubles.

So we were just building LLVM 3.3 from scratch. But this takes time.
Fortunately, we can package that manually-built version directly using
checkinstall.

This commit just uses LLVM 3.3 from a package created with checkinstall.
It contains the same version as if it were build manually.

This fixes issue #15.
